espa/metadata/metadata_api.py

```diff
@@ -22,19 +22,18 @@ class MetadataError(Exception):
 
 
 class Metadata(XMLInterface):
-    """Enhances the XMLInterface specifically for an ESPA Metadata XML
+    """Enhances the XMLInterface specifically for an XSD Schema defined XMLs
 
     Determines the best source for the schema to use for validation of the
-    specified ESPA Metadata XML.  First the ESPA_SCHEMA environment variable
-    is searched for and used if present.  Second the installation directory
-    where the schema should exist if installed on the local system.  And third
-    the URI where the schema should reside on the internet.
+    specified Metadata XML.  First environment variables are used in this
```

Comment:
Some rewording is needed.  Remove one of the "this"s in "used in this this order".  Also, the same sentence has "environment variables are used ... and used if present".  One of the uses of "used" probably should be removed or changed to a different word [maybe the first one should be "checked"].  Also, the sentence after that one is incomplete.